.storybook/preview-head.html

```diff
@@ -1,5 +1,8 @@
 <link href="https://fonts.googleapis.com/css?family=Roboto:100,200,300,400,500" rel="stylesheet" />
 <link href="http://fonts.cdnfonts.com/css/sofia-pro" rel="stylesheet" />
+<link href="https://fonts.googleapis.com/css2?family=Rubik:ital,wght@0,300;0,400;0,500;0,700;1,300;1,400;1,500;1,700&display=swap" rel="stylesheet">
```

Comment:
there is css variable which called font-family and other variables which using it (look in the typography.scss file).
the reason the font is updated is because we update monday-ui-style

Reply:
There are 2 `typography.scss` files in this repo, and I couldn't understand where the font is used exactly.

Can you point me to the specific line where it's used, or try to explain in more detail?

Reply:
sure.
there is 2 typography.scss files: one for the system and our components typography.scss and one for the storybook itself.
you can see in this file mixins we using our font variables https://github.com/mondaycom/monday-ui-react-core/blob/master/src/styles/typography.scss